riverdriver/river_driver_interface.go

```diff
@@ -118,6 +118,7 @@ type Executor interface {
 	JobInsertFast(ctx context.Context, params *JobInsertFastParams) (*rivertype.JobRow, error)
 	JobInsertFastMany(ctx context.Context, params []*JobInsertFastParams) (int, error)
 	JobInsertFull(ctx context.Context, params *JobInsertFullParams) (*rivertype.JobRow, error)
+	JobInsertManyReturning(ctx context.Context, params []*JobInsertFastParams) ([]*rivertype.JobRow, error)
```

Comment:
This one's still kind of a "job insert fast" variant, so that should probably still be in there.

I wonder too if we should try to match up which one gets the suffix between the client and driver a little bit more. In the client, the "fast" version gets a suffix, but in the driver here the "returning" variant gets the suffix (i.e. so they're flipped).

Maybe:

* `JobInsertFastMany` (called from `InsertManyFast`) -> `JobInsertFastManyNoReturning`
* `JobInsertManyReturning` (called from `InsertMany`) -> `JobInsertFastMany`

Reply:
Good call, renamed them as suggested :v: